docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc

```diff
@@ -0,0 +1,31 @@
+= Release notes - Payara Platform Development - Version 5.30.0
+
+== Supported APIs and Applications
+
+ * Java EE 8 Applications
+ * Jakarta EE 8
+ * MicroProfile 4.1
+
+### Bug Fixes
+* [FISH-5654] - NullPointerException when setting server log format to JSON
+* [FISH-1530] - Fix OpenAPI TCK Tag Collection Test
+* [FISH-1521] - Fix Incorrect Group/Role Mapping in OIDC Provider
+* [FISH-1520] - Fix MicroProfile JWT-Auth TCK Failures
+* [FISH-1312] - MP OpenAPI schema property hidden is ignored
+* [FISH-1204] - Missing example values in OpenApi components schema definition using nested objects
+* [FISH-1068] - Parameter Serialization error in open api document for query parameter
+* [FISH-1066] - ConcurrentModificationException is thrown for config key "org.eclipse.microprofile.config.ConfigValue:mp.config.profile:60s" on deploying REST application
+
+### Component Upgrade
+* [FISH-1504] - Upgrade MicroProfile JWT-Auth to 1.2.1
+* [FISH-786] - Integrate Security Connector with the Payara Platform
+
+### New Features
+* [FISH-1526] - Upgrade Payara Enterprise to MicroProfile 4.0
+* [FISH-1372] - MicroProfile Health 3.1
```

Comment:
Not sure this is worth mentioning? It's a part of MicroProfile 4.1 - it's not a standalone feature.